zerver/lib/message.py

```diff
@@ -27,15 +27,23 @@
     get_subscribed_stream_recipient_ids_for_user,
     num_subscribers_for_stream_id,
 )
-from zerver.lib.streams import can_access_stream_history, get_web_public_streams_queryset
+from zerver.lib.streams import (
+    UserGroupMembershipDetails,
```

Comment:
Probably `UserGroupMembershipDetails` should get moved to `zerver.lib.user_groups` rather than importing it from `streams.py` in other modules. Minor.